### ISSUE TITLE
base information about cbloss

### ISSUE THREAD

**Issue by Adorablepet** [vandit15/Class-balanced-loss-pytorch#7]
Thanks for sharing code.I have modified the code. I want to calculate the multivariate classification. The code is as follows. I can always report errors. I wonder if you have tried it?
```
pred = logits.log_softmax(dim=1)
cb_loss = F.cross_etropy(input=pred, target=labels, weights=weights)
```
Thanks.

**Comment by LcenArthas** [comment]
i have the same questios, have u sove it?

**Comment by Adorablepet** [comment]
@LcenArthas I have solved.
```
def forward(self, logits, labels):
      labels = labels.cuda()
      loss = F.cross_entropy(input=logits, target=labels, weight=self.weight)
      return loss
```
